code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -50,17 +50,16 @@
 	attacktext = "slashes"
 	attack_sound = 'sound/weapons/bladeslice.ogg'
 	status_flags = 0
+	maxHealth = 150
+	health = 150
 
 /mob/living/simple_animal/hostile/syndicate/melee/attackby(obj/item/O, mob/user, params)

```

Comment:
You can probably just remove this, without the block it's basically copypasta of the normal attackby.